tools/video_tools/pyscenedetect_shot_detection.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import json
+
+# Standard PySceneDetect imports:
+from scenedetect.video_manager import VideoManager
+from scenedetect.scene_manager import SceneManager
+# For caching detection metrics and saving/loading to a stats file
+from scenedetect.stats_manager import StatsManager
+
+# For content-aware scene detection:
+from scenedetect.detectors.content_detector import ContentDetector
+
+
+def main():
+    (input_file, threshold, output_json, output_csv) = sys.argv[1:5]
+    # Get a list of scenes as tuples (start, end) 
+    if threshold is None or isinstance(threshold, int) == False:
+        threshold = 30
+        print("Setting threshold to default because it wasn't a valid integer")
+
+    shots = find_shots(input_file, output_csv, threshold)
+
+    # Print for debugging purposes
+    for shot in shots:
+        print("start: " + str(shot[0]) + "  end: " + str(shot[1]))
```

Comment:
If you are using a recent python version you can also use f-strings. https://cito.github.io/blog/f-strings/
Maybe with:
`start, end = shots[:1]`

Reply:
Thank you for the suggestion, will keep that in mind